src/System.ComponentModel.Primitives/ref/System.ComponentModel.Primitives.cs

```diff
@@ -264,4 +269,191 @@ public sealed partial class RefreshPropertiesAttribute : System.Attribute
         public override bool IsDefaultAttribute() { throw null; }
     }
 }
-
+namespace System.ComponentModel.Design
+{
+    public partial interface IDesignerHost : System.ComponentModel.Design.IServiceContainer, System.IServiceProvider
```

Comment:
Why is this being brought down into primitives? If it is only in the implementation of FileSystemWatcher for the sync property then we shouldn't bring it down for that.